src/double_linked.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+
+class Node(object):
+    """Node class."""
+
+    def __init__(self, data=None, next_node=None):
+        """__init__ function."""
+        self.data = data
+        self.next_node = next_node
+        self.prev_node = None
+
+    def set_next(self, next_node):
+        self.next_node = next_node
+
+    def set_prev(self, prev_node):
+        self.prev_node = prev_node
+
+
+class DoubleLinkedList(object):
+    """Class for doube linked list."""
+
+    def __init__(self, input=None):
+        self.head = None
+        self.tail = None
+        if input is not None:
+            try:
+                [self.insert(x) for x in input]
```

Comment:
While it's cool that you've found a way to do this in a single line, it's kind of smelly and weird and clunky to use a list comprehension that creates a list full of `None`s which will be discarded right away. Also, your `input` symbol is overriding a default Python function name, which you may want to address.